scripts/bundleSize.sh

```diff
@@ -2,8 +2,8 @@
 
 # Size limit for all bundles used by each service (K)
 # Keep these +/- 5K and update frequently!
-min=605
-max=625
+min=500
```

Comment:
Do we really have a +/- 5K of 100K?!

Reply:
nope, I just put that as a placeholder for now. Happy to change to any suggestion.
![image](https://user-images.githubusercontent.com/30599794/61212963-9c40b800-a6fb-11e9-8311-f33fe7d3de96.png)

Reply:
Min needs to be 5K less than the smallest one, and Max 5K more than the biggest